bmwqemu.pm

```diff
@@ -70,6 +73,24 @@ our @ovmf_locations = (
 
 our %vars;
 
+sub result_dir { 'testresults' }
+
+sub logger {
```

Comment:
Now that this function exists, could `our $logger` be changed to `my $logger`?

Reply:
Well, `our $logger` so far is only overwritten in tests by giving it another logger instead. If we would replace that with calls to the method then we would first initialize a logger which we then overwrite with a new object. Not that nice by design. Unless we want to allow dependency injection by allowing an optional `$logger` argument here to have something akin to:

```perl
sub logger {
    $logger = shift // $logger // $log_module->new(level => 'debug', format => \&log_format_callback…
```